src/recyclerlistview/RecyclerListView.js

```diff
@@ -82,6 +82,9 @@ class RecyclerListView extends Component {
     }
 
     componentWillReceiveProps(newProps) {
+        if(!this._initComplete){
```

Comment:
What if the props have actually changed, things like layout provider etc? Simply returning here may lead to an inconsistent state